tests/cpptests/test_cpp_llapi.cpp

```diff
@@ -820,3 +820,14 @@ TEST_F(LLApiTest, testScorer) {
   RediSearch_ResultsIteratorFree(it);
   RediSearch_DropIndex(index);
 }
+
+TEST_F(LLApiTest, testStopwords) {
+  const char *words[] = {"Redis", "Labs"};
+  RSIndex* index = RediSearch_CreateIndex("index", NULL);
+  RediSearch_CreateStopwordsList(index, words, 2);
+  ASSERT_EQ(RediSearch_StopwordsList_Contains(index, words[0], strlen(words[0])), 1);
+  ASSERT_EQ(RediSearch_StopwordsList_Contains(index, words[1], strlen(words[1])), 1);
```

Comment:
Suggest to add another call to `RediSearch_StopwordsList_Contains` with a missing word and expect it to return `0`, also when Stopwords list was created properly, e.g., after line 829